pom.xml

```diff
@@ -103,6 +103,7 @@
         <version.semver>0.9.34-SNAPSHOT</version.semver>
         <version.springfox>2.9.2</version.springfox>
         <version.redline>1.2.9</version.redline>
+        <version.jdeb>1.8</version.jdeb>
```

Comment:
We don't need a property for this. Version properties should only be defined for dependencies and not plugins.

Reply:
I guess there are some plugins with versions that are bound to other dependencies, aren't they? I've spotted mix version convention handling, thus defined that one, I'll have it fixed.

Reply:
done